.github/workflows/ci.yml

```diff
@@ -24,7 +22,7 @@ jobs:
       - name: Set up Ruby ${{ matrix.ruby-version }}
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.7
+          ruby-version: 3.0
```

Comment:
If you can go to 3.1 that would be even better!

Reply:
done!